Dockerfile

```diff
@@ -15,13 +15,15 @@ FROM quay.io/konveyor/static-report as report
 
 FROM registry.access.redhat.com/ubi9/ubi-minimal
 ARG SEED_ROOT
+ARG HUB_VERSION=latest
```

Comment:
@fbladilo this is where we can just template `CI_VERSION`

Reply:
Just a _nit_, but it is the dockerfile for the hub so the HUB_ seems redundant.